supervised/utils/additional_metrics.py

```diff
@@ -175,6 +245,9 @@ def binary_classification(target, predictions, sample_weight=None):
         return {
             "metric_details": pd.DataFrame(details),
             "max_metrics": pd.DataFrame(max_metrics),
+            "accuracy_threshold_max_metric": pd.DataFrame(
```

Comment:
maybe change the name to `metrics_at_accuracy_threshold`

Reply:
`accuracy_threshold_max_metric` is different than `accu_threshold_details`.
Changed the later to `metrics_at_accuracy_threshold` as you told in the first comment